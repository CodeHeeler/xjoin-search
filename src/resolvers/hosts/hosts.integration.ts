import { getQuery } from '../../../test';

describe('hosts query', function () {
    test('fetch hosts', async () => {
        const data = await getQuery()({
            query: `
                { hosts(
                    filter:
                        {system_profile_fact:
                            {key: "os_release" value: "7.4"}
                        }
                    )
                    {
                        data {
                            id, account, display_name
                        }
                    }
                }`
        });

        expect(data).toMatchSnapshot();
    });
});
