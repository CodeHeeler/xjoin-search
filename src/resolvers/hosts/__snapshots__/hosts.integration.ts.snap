// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hosts query fetch hosts 1`] = `
Object {
  "data": Object {
    "hosts": Object {
      "data": Array [
        Object {
          "account": "901578",
          "display_name": "test01.rhel7.jharting.local",
          "id": "6e7b6317-0a2d-4552-a2f2-b7da0aece49d",
        },
      ],
    },
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
